fix: Add DIKO column and update modal with DIKO amount

Macxim opened the pull request:
Closes #383 

### Staking DIKO

<img width="1258" alt="Screen Shot 2021-11-26 at 4 03 24 PM" src="https://user-images.githubusercontent.com/1909957/143616646-4d9ad394-7a51-4a3a-b1eb-497fdb69961d.png">


### Unstake modal
<img width="545" alt="Screen Shot 2021-11-26 at 4 04 15 PM" src="https://user-images.githubusercontent.com/1909957/143616654-d59804cd-174a-4a64-9a97-9b31119fec0b.png">

Macxim commented:
Not sure if DIKO amount is correct or if there is a better way to calculate it. @nieldeckx @philipdesmedt

philipdesmedt commented on web/components/unstake-diko-modal.tsx:
```diff
@@ -107,7 +112,12 @@ export const UnstakeDikoModal = ({ showUnstakeModal, setShowUnstakeModal, staked
         })}{' '}
-        stDIKO.
+        stDIKO which equals to{' '}
+        {microToReadable(stakedAmount * stDikoToDiko).toLocaleString(undefined, {
```
I think this is fine, not sure if there can be slight rounding errors or an issue when a user unstakes & withdraws and gets less than the UI showed.... probably not?

nieldeckx reviewed (changes requested):
I don't think this is correct:
<img width="1316" alt="Screenshot 2021-11-29 at 08 58 35" src="https://user-images.githubusercontent.com/5592676/143829396-eb4fdac1-2ed2-4209-87dd-479c5addb7a3.png">
I have 10 stDIKO which should be equal to ~11.9 DIKO.

The function `get-stake-of` on the pool will return the amount of DIKO the user should get, not the amount of stDIKO. For the amount of stDIKO you can just use the state.balances.

Macxim commented:
@nieldeckx Agree. I had the same thought yesterday. I'll change it in a bit.

Macxim commented:
Fixed. What do you think @nieldeckx?

Updated screenshots below:

#### Staking DIKO

<img width="1262" alt="Screen Shot 2021-11-29 at 8 41 07 AM" src="https://user-images.githubusercontent.com/1909957/143835389-f981f588-1408-4aa9-985f-23979af50db4.png">

#### Unstake modal

<img width="547" alt="Screen Shot 2021-11-29 at 8 40 51 AM" src="https://user-images.githubusercontent.com/1909957/143835394-c83d322c-94c8-498d-9723-3476ba4e38e2.png">

nieldeckx commented:
@Macxim Perfect :)
